src/parsers/power_system_table_data.jl

```diff
@@ -492,10 +493,12 @@ function gen_csv_parser!(sys::System, data::PowerSystemTableData)
 end
 
 """
-Add loads to the System from the raw data.
+    busload_csv_parser!(sys::System, data::PowerSystemTableData)
+
+Add loads to the System from the raw bus data.
 
 """
-function load_csv_parser!(sys::System, data::PowerSystemTableData)
+function busload_csv_parser!(sys::System, data::PowerSystemTableData)
```

Comment:
I'm not sure why I haven't noticed this sooner...I do think this function has changed a lot over time.  Can't we create the PowerLoad directly in `bus_csv_parser!`?  We are running through redundant loops.